templates/mod/setenvif.conf.epp

```diff
@@ -18,17 +18,11 @@ BrowserMatch "JDK/1\.0" force-response-1.0
 BrowserMatch "Microsoft Data Access Internet Publishing Provider" redirect-carefully
 BrowserMatch "MS FrontPage" redirect-carefully
 BrowserMatch "^WebDrive" redirect-carefully
-BrowserMatch "^WebDAVFS/1.[0123]" redirect-carefully
-BrowserMatch "^gnome-vfs/1.0" redirect-carefully
+BrowserMatch "^WebDAVFS/1\.[012]" redirect-carefully
```

Comment:
Is it on purpose that 1.3 was dropped here?

Reply:
I think so: upstream switched from `1.[01234]` to `1\.[012]`:
https://svn.apache.org/viewvc/httpd/httpd/trunk/docs/conf/extra/httpd-dav.conf.in?r1=1915651&r2=1915650&pathrev=1915651